TestON/tests/PingallExample/PingallExample.params

```diff
@@ -1,12 +0,0 @@
-<PARAMS>
```

Comment:
Oh noes, I think it was my mistake, I think the one that got merge in some
commit was kaouther's pingallExample test so I was trying not to include it
in the commit I think I delete it in this commit by mistake.

On Fri, May 29, 2015 at 12:49 PM, Jon Hall <notifications@github.com> wrote:

> In TestON/tests/PingallExample/PingallExample.params
> <https://github.com/opennetworkinglab/ONLabTest/pull/123#discussion_r31358637>
> :
>
> > @@ -1,12 +0,0 @@
> > -<PARAMS>
>
> Did you mean to delete this example test?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opennetworkinglab/ONLabTest/pull/123/files#r31358637>.
>